chem/module_plumerise1.F

```diff
@@ -110,6 +110,8 @@ subroutine plumerise_driver (id,ktau,dtstep,                           &
                ebu(i,kts,j,p_ebu_no2)=ebu_in(i,1,j,p_ebu_in_no2)
                ebu(i,kts,j,p_ebu_co)=ebu_in(i,1,j,p_ebu_in_co)
                ebu(i,kts,j,p_ebu_co2)=ebu_in(i,1,j,p_ebu_in_co2)
+              ebu(i,kts,j,p_ebu_nh3) = ebu_in(i,1,j,p_ebu_in_nh3)
```

Comment:
For consistency, let's follow the indenting pattern, and also the "no space around the equal signs".

Reply:
Done

Reply:
line 113 needs to be indented, for consistency no spaces around the "=" sign, make it look the the original code; similar comments for line 115